test/unit/test_sap_cli_messageclass.py

```
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import json
+import unittest
+
+import sap.cli.messageclass
+
+from mock import Connection, Response, BufferConsole
+from infra import generate_parse_args
+from fixtures_sap_adt_messageclass import (
```

Flake8 (7.3.0)

[error] 'fixtures_sap_adt_messageclass.MESSAGE_CLASS_ADT_GET_RESPONSE_EMPTY_XML' imported but unused

(F401)